src/diffuse/meson.build

```diff
@@ -9,15 +9,42 @@ conf.set('VERSION', meson.project_version())
 conf.set('PKGDATADIR', pkgdatadir)
 conf.set('LOCALEDIR', join_paths(get_option('prefix'), get_option('localedir')))
 conf.set('SYSCONFIGDIR', join_paths(get_option('prefix'), get_option('sysconfdir')))
+conf.set('BINDIR', join_paths(get_option('prefix'), get_option('bindir')))
 
 configure_file(
   input: 'diffuse.in',
-  output: 'diffuse',
+  output: build_machine.system() == 'darwin' ? 'diffuse_impl' : 'diffuse',
   configuration: conf,
   install: true,
   install_dir: get_option('bindir')
 )
 
+if build_machine.system() == 'darwin'
+  install_subdir('mac-os-app/Diffuse.app', install_dir: '/Applications', strip_directory: false)
```

Comment:
This dir only contains the prebuilt icon, diffuse.icns. So this dir should perhaps go into data/icons instead, if we want to be strict...?

I put it here because I thought it would be cleaner and convenient to keep everything related to building the macOS app here, in one place... (Otherwise we will need an "if darwin" in data/icons's meson.build too, but that's perhaps no big deal?...)

Reply:
It's fine as it is (the data folder is more like a symmetry of the installed folders on Linux)